char_conv.c

```diff
@@ -116,7 +116,11 @@ EXPORT_DEF void hexify(const uint8_t *in, size_t in_length, char *out)
 #/* */
 static const uint8_t *get_char_gsm7_encoding(uint16_t c)
 {
-	int minor = c >> 8, major = c & 255;
+#if __BYTE_ORDER == __BIG_ENDIAN
+	int minor = c & 0xff, major = c >> 8;
+#else
+	int minor = c >> 8, major = c & 0xff;
+#endif
```

Comment:
Please separate these two while you're in the vicinity:
```
int minor = c & 0xff;
int major = c >> 8;
#else
int minor ...
int major ..
```
Multiple declarations and (worse) definitions are not good for readability.

Reply:
done